systems/plants/RigidBody.m

```diff
@@ -310,7 +310,11 @@ function dofnum(obj)
       % class. (maybe I should move it up to there?)
       fn = fieldnames(body.param_bindings);
       for i=1:length(fn)
-        body.(fn{i}) = double(subs(body.param_bindings.(fn{i}),poly,pval));
+        if isnumeric(pval)
+          body.(fn{i}) = double(subs(body.param_bindings.(fn{i}),poly,pval));
```

Comment:
yes, trying to substitute symbolic variables.

Here is the code snippet:
r1 = RigidBodyManipulator('tensionWParams.urdf');
q=msspoly('q',nq);
s=msspoly('s',nq);
c=msspoly('c',nq);
qt=TrigPoly(q,s,c);
qd=msspoly('qd',nq);
qdd=msspoly('qdd',nq);
u=msspoly('u',nu);
p=r1.getParamFrame.getPoly;
pr1 = setParams(r1,p);

Throws this error:
Error using msspoly/double (line 21)
Cannot cast non-constant msspoly to double.

Error in RigidBody/updateParams (line 314)
          body.(fn{i}) = double(subs(body.param_bindings.(fn{i}),poly,pval));

Error in RigidBodyManipulator/applyToAllRigidBodyElements (line 2441)
        model.body(i) = feval(fcn,model.body(i),varargin{:});

Error in RigidBodyManipulator/setParams (line 1064)
      model = applyToAllRigidBodyElements(model,'updateParams',fr.getPoly,p);

Error in extractDynParams (line 16)
pr1 = setParams(r1,p);